frontend/src/views/SettingWorkspaceSubscription.vue

```diff
@@ -45,12 +45,56 @@
         </dt>
         <dd class="mt-1 text-4xl">{{ instanceCount }}</dd>
       </div>
-      <div class="my-3">
+      <div v-if="!subscriptionStore.isFreePlan" class="my-3">
         <dt class="text-gray-400">
           {{ $t("subscription.expires-at") }}
         </dt>
         <dd class="mt-1 text-4xl">{{ expireAt || "n/a" }}</dd>
       </div>
+      <div
+        v-if="subscriptionStore.currentPlan != PlanType.ENTERPRISE"
```

Comment:
Maybe needs to add a `canTrial`?

```suggestion
        v-if="canTrial && subscriptionStore.currentPlan != PlanType.ENTERPRISE"
```

Reply:
Nice catch